core/field_textinput.js

```diff
@@ -234,16 +260,14 @@ Blockly.FieldTextInput.prototype.bindEvents_ = function(htmlInput) {
   htmlInput.onKeyDownWrapper_ =
       Blockly.bindEventWithChecks_(
           htmlInput, 'keydown', this, this.onHtmlInputKeyDown_);
-  // Bind to keyup -- trap Enter; resize after every keystroke.
```

Comment:
What about Enter?

Reply:
As far as I can tell onHtmlInputChange_ doesn't do anything with enter. I think this was just a typo so I removed it.